### ISSUE TITLE
git hook - persist notebooks giving a weird error - no notebooks in dir

### ISSUE THREAD

**Issue by tanyav1995** [bsnacks000/datascience-project#7]
```
Running pre-commit hook...
Traceback (most recent call last):
  File "/home/tanya/ds-projects/rtm-unc-new/venv/bin/persist-notebooks", line 11, in <module>
    load_entry_point('rtm-unc-new', 'console_scripts', 'persist-notebooks')()
  File "/home/tanya/ds-projects/rtm-unc-new/venv/lib/python3.6/site-packages/pkg_resources/__init__.py", line 480, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/tanya/ds-projects/rtm-unc-new/venv/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2693, in load_entry_point
    return ep.load()
  File "/home/tanya/ds-projects/rtm-unc-new/venv/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2324, in load
    return self.resolve()
  File "/home/tanya/ds-projects/rtm-unc-new/venv/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2330, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/tanya/ds-projects/rtm-unc-new/_rtm_unc_new_build_system/reports.py", line 24
                    Remove spaces from the filename to process or process manually.""")
SyntaxError: f-string: empty expression not allowed
```

**Comment by bsnacks000** [comment]
Ok this is an easy one... just a dumb mistake...

**Comment by bsnacks000** [comment]
Fixes the SyntaxError 887dccd

nbconvert was having trouble with filepaths with spaces so I had added this warning here
